wrench/src/yaml_frame_reader.rs

```diff
@@ -464,7 +464,19 @@ impl YamlFrameReader {
             return None;
         }
 
-        let file = rsrc_path(&item["image"], &self.aux_dir);
+        //rsrc_path(&item["image"], &self.aux_dir);
+        let file = match item["image"].as_str() {
+            Some(filename) => {
+                let mut file = self.aux_dir.clone();
+                file.push(filename);
+                file
+            }
+            None => {
+                //warn!("No image provided for the image-mask!");
```

Comment:
This warning looks useful, but if you still think it doesn't belong, might as well remove the line.

Reply:
commented out because wrench isn't hooked up to log. I suppose this fixable :)

Reply:
Hrm. I notice that when I run the ahem.yaml reftest I get this output:
```
REFTEST reftests/text/ahem.yaml == reftests/text/ahem-ref.yaml
Skipping unknown item type: Hash({String("bounds"): Array([Integer(0), Integer(0), Integer(0), Integer(0)]), String("clip-rect"): Array([Integer(0), Integer(0), Integer(0), Integer(0)]), String("clip-and-scroll"): Integer(0), String("backface-visible"): Boolean(true)})
Skipping unknown item type: Hash({String("bounds"): Array([Integer(0),
```